projects/1.20/assets/netherific/nourished_nether/lang/zh_cn.json

```
@@ -0,0 +1,151 @@
+{
+  "advancements.full_necronium_hoe_advancement.descr": "用死灵锄击杀足够的生物，将其升至3级",
+  "advancements.full_necronium_hoe_advancement.title": "时间花得值",
+  "advancements.nether_delight.descr": "合成并食用幽灵猪肉",
+  "advancements.nether_delight.title": "诡异美味",
+  "advancements.nourished_nether_root.descr": "更火辣的下界在等待……",
+  "advancements.nourished_nether_root.title": "丰饶下界",
+  "advancements.post_mortem_advancement.descr": "在死后图腾效果下重生",
+  "advancements.post_mortem_advancement.title": "它又活了……",
+  "advancements.soul_forge_advancement.descr": "用灵魂熔炉将金锭锻造成死灵锭",
+  "advancements.soul_forge_advancement.title": "非生之金属",
+  "advancements.soul_quartz_advancement.descr": "用锄头耙取富灵魂泥",
```

```suggestion
  "advancements.soul_quartz_advancement.descr": "用锄头耙取富灵魂泥浆",
```
`富灵魂泥浆`统一称呼
以及这里的`Rake`是指用锄破坏，还是类似于用锄开垦